src/mixins/crownConfig.js

```diff
@@ -107,6 +107,7 @@ export default {
       }
 
       this.$emit('add-node', {
+        id: 'sequence-flow-button',
```

Comment:
This can be removed; `addNode` doesn't do anything with `id`, and the "add sequence flow" buttons for task, start event and gateway already have IDs set.